.github/workflows/checkLicenses.yml

```diff
@@ -24,7 +24,7 @@ jobs:
       - name: Set up Go 1.20
         uses: actions/setup-go@v3
         with:
-          go-version: '>=1.20.5'
```

Comment:
We might be able to use 'stable' here (for things where we are running go but not necessarily building artifacts): https://github.com/actions/setup-go#using-stableoldstable-aliases.  But OTOH that might be _more_ confusing than just having one, explicitly specified version.

Reply:
Yeah, I agree. But we have had quite a few CVE's come up about go, so on the other hand it is useful to be explicit about which version we are using.